pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -259,6 +262,7 @@ func (b *bucket) aggregationToPayloads() []*pb.ClientStatsPayload {
 				HTTPStatusCode: aggrKey.StatusCode,
 				Type:           aggrKey.Type,
 				Synthetics:     aggrKey.Synthetics,
+				PeerTags:       counts.peerTags,
```

Comment:
We'll need a hash on the `aggrKey` to make this work. I think i'm beginning to see why you wanted to leave this out...